Source/utils/paths.cpp

```diff
@@ -1,6 +1,8 @@
 #include "utils/paths.h"
 #include "utils/log.hpp"
 
+#include <optional>
+
 #include <SDL.h>

```

Comment:
Fix include order:

1. corresponding header
2. stdlib
3. system dependencies
4. our code

```suggestion
#include "utils/paths.h"

#include <optional>

#include <SDL.h>

#include "utils/log.hpp"
```